packages/wrangler/src/dev.tsx

```diff
@@ -581,6 +581,61 @@ export async function startDev(args: StartDevOptions) {
 					);
 				});
 			}
+
+			if (process.stdout.isTTY && (args.showInteractiveDevSession ?? true)) {
+				const unregisterHotKeys = registerHotKeys([
+					{
+						keys: ["b"],
+						label: "open a browser",
+						handler: async () => {
+							const { url } = await devEnv.proxy.ready.promise;
+							await openInBrowser(url.href);
+						},
+					},
+					{
+						keys: ["d"],
+						label: "open devtools",
+						handler: async () => {
+							const { inspectorUrl } = await devEnv.proxy.ready.promise;
+
+							// TODO: refactor this function to accept a whole URL (not just .port and assuming .hostname)
+							await openInspector(
+								parseInt(inspectorUrl.port),
+								devEnv.config.latestConfig?.name
+							);
+						},
+					},
+					{
+						keys: ["l"],
+						label: () =>
+							`turn ${devEnv.config.latestConfig?.dev?.remote ? "on" : "off"} local mode`,
+						handler: async () => {
+							await devEnv.config.patch({
+								dev: {
+									...devEnv.config.latestConfig?.dev,
+									remote: !devEnv.config.latestConfig?.dev?.remote,
+								},
+							});
+						},
+					},
```

Comment:
This should only be added if `forceLocal` is false

Reply:
`wrangler dev` doesn't support `forceLocal`. When this is implemented for `wrangler pages dev`, we would simply not have this option (or if it was abstracted to work for both, we'd implement forceLocal then)

Reply:
`wrangler dev` doesn't, but this code path is also used for `wrangler pages dev` (they both use the `startDev` entry)